Aptoide/aptoide-client-v8
# [feature/ASV-1445] APPC App View enhancement

**Note: This PR is currently on standby because of SVG/PNG discussions, and it should not be accepted atm.**

**What does this PR do?**

   As described by the title, this PR "enhances" the app view for APPC apps both visually and functionally. For visuals, it changes the Install button background, the progress bar background, the collapsible toolbar background (plus animations) and removes the "Flag this app" section completely. Functionally, it removes every ad from the app view, i.e., banners, native ads and interstitials and it also removes the "Share this app" dialog.

As a side note, it also changes the appview background back to white.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Check if everything works as expected, i.e. if appc apps do show all the mentioned visuals and if ads, share dialog and "Flag this app" section do not show up. Also check if animations are ok in different devices and if performance is fine.

**What are the relevant tickets?**

  [ASV-1445](https://aptoide.atlassian.net/browse/ASV-1445)

**Code Review Checklist**
